tests/general/IndentationBearTest.py

```diff
@@ -130,7 +130,7 @@ def test_blank_lines(self):
                       "\tIndents even after blank line}")
         invalid_file = ("{ trying indent",
                         "\n",
-                        "should've Indented after blank line}")
+                        "should ave Indented after blank line}")
```

Comment:
But these aren't single quotes in double quotes! , These are just single quotes which do not end in that line!